src/main/java/perkpack/controllers/CardController.java

```diff
@@ -1,60 +1,34 @@
 package perkpack.controllers;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 import perkpack.models.Card;
 import perkpack.repositories.CardRepository;
-import javax.validation.Valid;
 
-@RestController
-@RequestMapping("cards")
+@Controller
```

Comment:
As discussed earlier, classes with @Controller annotations will be removed in milestone 2. We will be moving to @RestController exclusively.